Source/Csla/Server/FactoryDataPortal.cs

```diff
@@ -17,43 +17,24 @@ namespace Csla.Server
   /// invokes an object factory rather than directly
   /// interacting with the business object.
   /// </summary>
-  public class FactoryDataPortal : IDataPortalServer, Core.IUseApplicationContext
+  public class FactoryDataPortal : IDataPortalServer
   {
-    ApplicationContext Core.IUseApplicationContext.ApplicationContext { get => ApplicationContext; set => ApplicationContext = value; }
-    private ApplicationContext ApplicationContext { get; set; }
-
-    #region Factory Loader
-
-    private static IObjectFactoryLoader _factoryLoader;
-
     /// <summary>
-    /// Gets or sets a delegate reference to the method
-    /// called to create instances of factory objects
-    /// as requested by the ObjectFactory attribute on
-    /// a CSLA .NET business object.
+    /// Creates an instance of the type.
     /// </summary>
-    public static IObjectFactoryLoader FactoryLoader
+    /// <param name="applicationContext"></param>
+    /// <param name="factoryLoader"></param>
+    /// <param name="inspector"></param>
+    public FactoryDataPortal(ApplicationContext applicationContext, IObjectFactoryLoader factoryLoader, IDataPortalExceptionInspector inspector)
     {
-      get
-      {
-        if (_factoryLoader == null)
-        {
-          string setting = ConfigurationManager.AppSettings["CslaObjectFactoryLoader"];
-          if (!string.IsNullOrEmpty(setting))
-            _factoryLoader =
-              (IObjectFactoryLoader)Activator.CreateInstance(Type.GetType(setting, true, true));
-          else
-          _factoryLoader = new ObjectFactoryLoader();
-        }
-        return _factoryLoader;
-      }
-      set
-      {
-        _factoryLoader = value;
-      }
+      ApplicationContext = applicationContext;
+      FactoryLoader = factoryLoader;
+      Inspector = inspector;
     }
 
-    #endregion
+    private ApplicationContext ApplicationContext { get; set; }
+    private IObjectFactoryLoader FactoryLoader { get; set; }
+    private IDataPortalExceptionInspector Inspector { get; set; }
```

Comment:
The IDataPortalExceptionInspector property was called ExceptionInspector in the DataPortal class; might it be more consistent if we named it that here, too?